DigitalOcean.API/Clients/IDropletActionsClient.cs

```diff
@@ -43,7 +43,7 @@ public interface IDropletActionsClient {
         /// A Droplet restoration will rebuild an image using a backup image. The image ID that is passed in must be a backup of
         /// the current Droplet instance. The operation will leave any embedded SSH keys intact.
         /// </summary>
-        Task<Action> Restore(int dropletId, int imageId);
+        Task<Action> Restore(int dropletId, object imageIdOrSlug);
```

Comment:
API method can allow both imageId integer and slug string. Same as `Requests.Droplet`.`ImageIdOrSlug` or  `IImagesClient.Get(object imageIdOrSlug)` or `IDropletActionsClient.Rebuild(int dropletId, object imageIdOrSlug)`.